src/main/java/com/twilio/oai/TwilioJavaGenerator.java

```diff
@@ -301,6 +316,22 @@ private enum Operation {
 
         private final String prefix;
     }
+    private ArrayList<List<CodegenParameter>> generateSignatureList(final Map<String, Object> resource, final CodegenOperation co, boolean isVersionV2010) {
```

Comment:
keep track of signature list that would contain different combinations of signatures for constructor generation (since Account sid is now optional, so different constructor are needed)

Reply:
Should make this a comment on the method.